src/PaymentGateways/Gateways/Stripe/Actions/UpdateStripeFormBuilderSettingsMeta.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Give\PaymentGateways\Gateways\Stripe\Actions;
+
+use Give\DonationForms\Models\DonationForm;
+
+/**
+ * Update Stripe settings on backwards compatible form meta.
+ *
+ * @since 3.0.0
+ */
+class UpdateStripeFormBuilderSettingsMeta
+{
+    /**
+     * @since 3.0.0
+     * @param  DonationForm  $form
+     */
+    public function __invoke(DonationForm $form)
+    {
+        $paymentGatewaysBlock = $form->blocks->findByName('givewp/payment-gateways');
+        if (!$paymentGatewaysBlock) {
+            give()->form_meta->delete_meta($form->id, 'give_stripe_per_form_accounts');
+            give()->form_meta->delete_meta($form->id, '_give_stripe_default_account');
+
+            return;
+        }
```

Comment:
Is there a reason we need to delete this meta?

Reply:
Not really. Removing it.

Reply:
Resolved: https://github.com/impress-org/givewp/pull/6903/commits/b6616ec12999d1cff98f6985b472f994a7e4da1b